src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -175,18 +175,23 @@ public void onActivityCreated(Activity activity, Bundle bundle) {
 
         @Override
         public void onActivityStarted(Activity activity) {
+            ConnectivityIndicator.start(mContext);
```

Comment:
I don't see why we need to initialize `ConnectivityIndicator` every time an activity starts. It should only be initialized when we need to use it. (i.e. `onActivityResumed` if we are on edit mode and want to show the indicator). `ConnectivityIndicator.getInstance.show()` when the app resumes seems like a better approach.